lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -11,6 +11,23 @@ include_once 'includes/KudosController.php';
 include_once 'includes/KudosTransformer.php';
 
 
+function dosomething_kudos_menu() {
+  $items = array();
+
+  // Admin Kudos configuration page.
+  $items['admin/config/dosomething/dosomething_kudos'] = array(
```

Comment:
instead of making this a new form, I think it makes sense to bundle this under the `/admin/config/dosomething/dosomething_reportback` config menu